swift/extractor/Configuration.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SWIFT_EXTRACTOR_CONFIGURATION_H_
+#define SWIFT_EXTRACTOR_CONFIGURATION_H_
+
+#include <string>
+#include <vector>
+
+namespace codeql {
+struct Configuration {
```

Comment:
On one hand, we are under the `swift` folder so that should be enough.
OTOH, prefixed names are better for "greppability."
I'm fine with either way, @redsun82 do you have any thoughts on this?

Reply:
> we are under the `swift` folder so that should be enough.

That doesn't affect linkage though. We've spoken about sharing some code between the C++ and Swift extractors in the future. If that happens, there's a clear possibility that the shared CodeQL code have a CodeQL config in a `codeql::Configuration` class. Same goes for if the C++ extractor also went for an "under a `c` folder" argument and named its own config the same thing.

> OTOH, prefixed names are better for "greppability."

Which is also important for "modern", web-based flows such as cs.github.com and PR reviews. And even in an IDE, if I'm switching back and forth, it's nice to know at a glance what I'm looking at.

Reply:
I'm ok with suffixing this with swift

Reply:
Ok, let's prefix it then.